osm2streets/src/lanes/placement.rs

```diff
@@ -23,66 +22,60 @@ impl TryFrom<&str> for RoadPosition {
                             "left_of" => Ok(LeftOf(LtrLaneNum::Forward(lane))),
                             "middle_of" => Ok(MiddleOf(LtrLaneNum::Forward(lane))),
                             "right_of" => Ok(RightOf(LtrLaneNum::Forward(lane))),
-                            _ => Err(()),
+                            _ => Err(anyhow!("unknown lane position specifier: {kind}")),
```

Comment:
You may also be able to use the `bail!` macro here, which does `return Err(anyhow!(...))`